### ISSUE TITLE
Add interlinear words in input area

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/introdemo#13]
Inside the #input div, between the transcription and the translation, we should have the tokenized words automatically populate, using a very simple template. Each word will consist of a token, which is uneditable and automatically populated by the automatically-tokenized transcription, and a gloss, which *is* editable by the user. Then when the gloss is edited, the JSON display should update accordingly.

Basically @jmangel and I realized that, in order for the interlinear gloss visualization to be useful, users have to be able to input 3 things: the transcription, the translation, and the glosses for each word. So the input area needs to be updated to allow for the glossing of words.

**Comment by dwhieb** [comment]
![win_20150130_205054](https://cloud.githubusercontent.com/assets/5892997/5986841/c9776c66-a8c1-11e4-9d7e-93ead208bb48.JPG)

We thought it should look roughly like this, with the middle area auto-populating based on a tokenization of the transcription, using an html template

**Comment by dwhieb** [comment]
Fixed by e7101150920d6ce3e004678aefcb154660365f91
